environment.yml

```diff
@@ -3,65 +3,60 @@ channels:
   - conda-forge
 dependencies:
   - python >=3.10
-  - pip
-  - numpy
-  - scipy
-  - openblas
-  - matplotlib
-  - tqdm
-  - pooch>=1.5
+  - antio >=0.4.0
+  - darkdetect
   - decorator
-  - h5io
-  - packaging
-  - numba
-  - pandas
-  - pyarrow
-  - xlrd
-  - scikit-learn
+  - defusedxml
+  - dipy
+  - edfio >=0.2.1
+  - eeglabio
+  - h5io >=0.2.4
   - h5py
-  - jinja2
-  - pillow
-  - statsmodels
-  - jupyter
-  - ipython !=8.7.0
-  - joblib
-  - psutil
-  - numexpr
-  - imageio
-  - spyder-kernels >=1.10.0
   - imageio >=2.6.1
   - imageio-ffmpeg >=0.4.1
-  - vtk >=9.2
-  - traitlets
-  - pyvista >=0.32,!=0.35.2,!=0.38.0,!=0.38.1,!=0.38.2,!=0.38.3,!=0.38.4,!=0.38.5,!=0.38.6,!=0.42.0
-  - pyvistaqt >=0.4
-  - qdarkstyle !=3.2.2
-  - darkdetect
-  - dipy
+  - ipyevents
+  - ipympl
+  - ipython !=8.7.0
+  - ipywidgets
+  - jinja2
+  - joblib
+  - jupyter
+  - lazy_loader >=0.3
+  - matplotlib >=3.6
+  - mffpy >=0.5.7
+  - mne-qt-browser
   - nibabel
-  - openmeeg >=2.5.5
   - nilearn
+  - numba
+  - numpy >=1.23,<3
+  - openmeeg >=2.5.5
+  - packaging
+  - pandas
+  - pillow
+  - pip
+  - pooch >=1.5
+  - pyarrow
+  - pybv
+  - pymatreader
+  - PySide6 !=6.8.0,!=6.8.0.1
+  - python-neo
   - python-picard
+  - pyvista >=0.32,!=0.35.2,!=0.38.0,!=0.38.1,!=0.38.2,!=0.38.3,!=0.38.4,!=0.38.5,!=0.38.6,!=0.42.0
+  - pyvistaqt >=0.4
+  - qdarkstyle !=3.2.2
   - qtpy
-  - pyside6 !=6.8.0,!=6.8.0.1
-  - mne-base
-  - seaborn-base
-  - mffpy >=0.5.7
-  - ipyevents
-  - ipywidgets
-  - ipympl
+  - scikit-learn
+  - scipy >=1.9
+  - sip
+  - snirf
+  - statsmodels
+  - threadpoolctl
+  - tqdm
+  - traitlets
   - trame
   - trame-vtk
   - trame-vuetify
-  - jupyter_client
-  - nbformat
-  - nbclient
-  - mne-qt-browser
-  - pymatreader
-  - eeglabio
-  - edfio >=0.2.1
-  - pybv
-  - mamba
-  - lazy_loader
-  - defusedxml
-  - python-neo
+  - vtk >=9.2
+  - xlrd
+  - pip:
+      - pyobjc-framework-Cocoa >=5.2.0;platform_system=='Darwin'
```

Comment:
For now then maybe we just exclude this bit in your code with a `TODO` comment that we should add it back in? FWIW I don't think this module meaningfully affects our tests so no big problem to omit it for a while

Reply:
yeah I guess so. I suspect you're right that it won't affect our tests, but it is also a user-facing document so I'm inclined not to put an incomplete version in `main`... but in this case since (IIUC) all it does is set the app name in the Qt window title bar, the stakes are pretty low.

Reply:
done in 32d5768dd01155b49760556004fb6f15e447bc0c